chain/ethereum/src/trigger.rs

```diff
@@ -145,32 +146,32 @@ impl blockchain::MappingTrigger for MappingTrigger {
                 params,
                 handler: _,
             } => {
-                if heap.api_version() >= Version::new(0, 0, 2) {
+                let ethereum_data = EthereumEventData {
```

Comment:
As we talked, maybe it's worth changing this to `ethereum_event_data` or just `event_data` 🙂